cmd/state/main.go

```diff
@@ -182,7 +182,7 @@ func run(args []string, isInteractive bool, out output.Outputer) error {
 
 	if childCmd != nil && !childCmd.SkipChecks() {
 		// Auto update to latest state tool version, only runs once per day
-		if updated, err := autoUpdate(args, out, pjPath); err != nil || updated {
+		if updated, err := autoUpdate(args, cfg, out, pjPath); err != nil || updated {
```

Comment:
If this is cheaper to do for the release branch then I'm on board but I'd prefer we don't pass the cfg around so much and instead pass around the thing we actually want, ie. the tag.